server/apps/frontend/src/Seller/SellerView/SellerView.tsx

```diff
@@ -0,0 +1,110 @@
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  Legend,
+  LineElement,
+  LinearScale,
+  PointElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import '@formatjs/intl-numberformat/polyfill';
+import '@formatjs/intl-numberformat/locale-data/fr';
+import { SalesHistory, Seller } from '../Seller.hook';
+import { getDataHistory, options, stringToColor } from './SellerView.hook';
```

Comment:
A good rule of thumb is to have you `import 'xxx'` either first or last, not mixed in between imports.

```suggestion
import { SalesHistory, Seller } from '../Seller.hook';
import { getDataHistory, options, stringToColor } from './SellerView.hook';
import '@formatjs/intl-numberformat/polyfill';
import '@formatjs/intl-numberformat/locale-data/fr';
```